### ISSUE TITLE
systemd-rc-local-generator[4424]: /etc/rc.local is not executable: Permission denied

### ISSUE THREAD

**Issue by mbiebl** [systemd/systemd#11044]
**systemd version the issue has been seen with**

git master (v239-3046-g3a5d89fa8)

**Used distribution**
 
Debian sid

During boot (and on every daemon-reload) I get the following error in the journal:
```
Dez 04 14:11:00 pluto systemd-rc-local-generator[4424]: /etc/rc.local is not executable: Permission denied
```
/etc/rc.local is indeed not executable:
```
# ls -al /etc/rc.local 
-rw-r--r--. 1 root root 388 Apr 22  2016 /etc/rc.local
```
That the executable bit is not set, does not indicate an error but is a supported configuration, so no error should be logged in this case.

This does not happen with v239 and seems a recent regression.

**Comment by poettering** [comment]
See #10955. Also:  https://bugzilla.redhat.com/show_bug.cgi?id=1516188

I figure a compromise is to downgrade it to LOG_INFO or LOG_NOTICE...

@keszybz opinion?

**Comment by mbiebl** [comment]
Hm, ok. I guess if the error message was more explicit, it would be clear why it is logged in the first place.
To me it looked like an unhandled error condition.
With the reference to the rhbz bug report, I can understand why it is logged, but the error message should be more obvious in that.

**Comment by mbiebl** [comment]
Maybe something like "Skipping execution of rc-local.service as /etc/rc.local is not marked as executable"

**Comment by poettering** [comment]
Fix waiting in #11053
